Assets/MRTK/StandardAssets/Shaders/MixedRealityStandard.shader

```diff
@@ -115,7 +115,7 @@ Shader "Mixed Reality Toolkit/Standard"
         Pass
         {
             Name "Main"
-            Tags{ "RenderType" = "Opaque" "LightMode" = "ForwardBase" }
+            Tags{ "RenderType" = "Opaque" "LightMode" = "LightweightForward" }
```

Comment:
Could you change this line so that the shader works with the built-in render pipeline by default?
```suggestion
            Tags{ "RenderType" = "Opaque" "LightMode" = "ForwardBase" }
```